gallery/visualtests/nxDrawer.js

```diff
@@ -0,0 +1,168 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+const { setupBrowser } = require('./testUtils');
+
+describe('NxDrawer', function() {
+  const {
+    waitAndGetElements,
+    checkScreenshot,
+    getPage,
+    a11yTest,
+    wait
+  } = setupBrowser('#/pages/Drawer');
+
+  async function openDrawer(drawerId, buttonId) {
+    const openDrawerBtnSelector = `#${buttonId}`;
+
+    const [openDrawerBtn] = await waitAndGetElements(openDrawerBtnSelector);
+
+    await openDrawerBtn.click();
+
+    await wait(200);
+
+    const [drawer] = await waitAndGetElements(`#${drawerId}`);
+
+    return drawer;
+  }
+
+  function simpleDrawerTest(drawerId, buttonId) {
+    return async function() {
+      const drawer = await openDrawer(drawerId, buttonId);
+
+      await checkScreenshot(drawer);
+    };
+  }
+
+  function drawerA11yTest(drawerId, buttonId) {
+    return async function() {
+      await openDrawer(drawerId, buttonId);
+      await a11yTest();
+    };
+  }
+
+  beforeEach(async function() {
+    await getPage().setViewport({ width: 1366, height: 1000 });
+  });
+
+  describe('Simple NxDrawer', function() {
+    it('looks right', simpleDrawerTest('nx-drawer-simple', 'nx-drawer-simple-open-button'));
+
+    it('passes a11y checks', drawerA11yTest('nx-drawer-simple', 'nx-drawer-simple-open-button'));
+  });
+
+  describe('Narrow NxDrawer', function() {
+    it('looks right', simpleDrawerTest('nx-drawer-narrow', 'nx-drawer-narrow-open-button'));
+  });
+
+  describe('NxDrawer With Footer', function() {
+    it('looks right', simpleDrawerTest('nx-drawer-with-footer', 'nx-drawer-with-footer-open-button'));
+  });
+
+  describe('NxDrawer With Subtitle', function() {
+    it('looks right', simpleDrawerTest('nx-drawer-with-subtitle', 'nx-drawer-with-subtitle-open-button'));
+  });
+
+  describe('NxDrawer With Paragraph', function() {
+    it('looks right', simpleDrawerTest('nx-drawer-with-paragraph', 'nx-drawer-with-paragraph-open-button'));
+  });
+
+  describe('NxDrawer With Subtitle and Paragraph', function() {
+    it('looks right', simpleDrawerTest('nx-drawer-with-subtitle-and-paragraph',
+        'nx-drawer-with-subtitle-and-paragraph-open-button'));
+  });
+
+  describe('NxDrawer + NxDropdown ESC Closing Behavior', function() {
+    async function isFocused(el) {
```

Comment:
This is available in `testUtils`

Reply:
> This is available in `testUtils`

Fixed here: 8356f233f575973b8be5be477b77d35da0b492c6